android/src/main/java/com/github/godness84/RNRecyclerViewList/RecyclerViewBackedScrollView.java

```diff
@@ -452,6 +452,11 @@ public int calculateDtToFit(int viewStart, int viewEnd, int boxStart, int boxEnd
         this.getLayoutManager().startSmoothScroll(smoothScroller);
     }
 
+    public void setInverted(boolean inverted) {
+        LinearLayoutManager layoutManager = (LinearLayoutManager) getLayoutManager();
```

Comment:
Are you sure this is always a `LinearLayoutManager`? 
I mean, on the chat, yes - but on the context of the upstream PR, I don't think you can make this assumption.